mysql_fdw.c

```diff
@@ -942,11 +950,15 @@ mysqlGetForeignPlan(PlannerInfo *root,RelOptInfo *baserel, Oid foreigntableid, F
 	 * field of the finished plan node; we can't keep them in private state
 	 * because then they wouldn't be subject to later planner processing.
 	 */
-	return make_foreignscan(tlist,
-							local_exprs,
-							scan_relid,
-							params_list,
-							fdw_private);
+	return make_foreignscan(tlist
```

Comment:
I have yet to be able to reproduce the crash that you're reporting (I've admittedly done very limited testing). Could you please provide the SQL that you used when you encountered the crash (on both the MySQL and PostgreSQL sides)? Could you also please provide the PostgreSQL commit that you were on? Any other details about your build environment would be appreciated.